templates/forgejo.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Forgejo</Name>
+  <Repository>codeberg.org/forgejo/forgejo:1.18</Repository>
+  <Registry>https://codeberg.org/forgejo/-/packages/container/forgejo/versions</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Date>2023-01-02</Date>
+  <Changes>
+- Initial Release
+  </Changes>
+  <Support>https://forums.unraid.net/topic/87798-support-selfhostersnets-template-repository</Support>
+  <Overview>
+  Forgejo is a self-hosted lightweight software forge. Easy to install and low maintenance, it just does the job.[br]
+  Brought to you by an inclusive community under the umbrella of Codeberg e.V., a democratic non-profit organization, Forgejo can be trusted to be exclusively Free Software. It is a "soft" fork of Gitea with a focus on scaling, federation and privacy.</Overview>
+  <Category>Backup: Cloud: Productivity: Tools:</Category>
+  <WebUI>http://[IP]:[PORT:3000]</WebUI>
+  <TemplateURL/>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/forgejo.png</Icon>
+  <ExtraParams/>
+  <Networking>
+    <Mode>bridge</Mode>
+    <Publish>
+      <Port>
+        <HostPort>22</HostPort>
+        <ContainerPort>22</ContainerPort>
+        <Protocol>tcp</Protocol>
+      </Port>
+      <Port>
+        <HostPort>3000</HostPort>
+        <ContainerPort>3000</ContainerPort>
+        <Protocol>tcp</Protocol>
+      </Port>
+    </Publish>
+  </Networking>
+  <Data>
+    <Volume>
+      <HostDir>/mnt/user/appdata/forgejo</HostDir>
+      <ContainerDir>/data</ContainerDir>
+      <Mode>rw</Mode>
+    </Volume>
+  </Data>
```

Comment:
https://selfhosters.net/docker/templating/templating/#114-shave-off-the-xml

Reply:
Thanks @Roxedus, I've updated it after reading through that page.